.travis.yml

```diff
@@ -1,8 +1,10 @@
 language: ruby
 rvm:
-  - 2.3.0
-  - 2.4.0
-  - 2.5.0
+  - 2.3.8
+  - 2.4.10
+  - 2.5.8
+  - 2.6.6
+  - 2.7.1
```

Comment:
If we want to test against the latest rubies, would it work to just omit the patch level here? Or would it include unstable rubies?

Reply:
Not sure, I will test it out and see what it does

Reply:
Looks like it does work, it is aliasing the latest stable minor version

Reply:
Nice!